Cargo.toml

```
@@ -32,7 +32,7 @@ thread_local = { version = "1.1.0", optional = true }
 [dev-dependencies]
 once_cell = "1.9.0"
 version-sync = "0.9.2"
-bevy = { version = "0.8", default-features = false, features = [
+bevy = { git = "https://github.com/bevyengine/bevy", branch = "main", default-features = false, features = [
```

```suggestion
bevy = { version = "0.9", default-features = false, features = [
```

Bevy 0.9 is now released. :)